frontend/packages/console-shared/src/components/spotlight/Spotlight.tsx

```diff
@@ -8,7 +8,18 @@ type SpotlightProps = {
 };
 
 const Spotlight: React.FC<SpotlightProps> = ({ selector, interactive }) => {
-  const element = React.useMemo(() => document.querySelector(selector), [selector]);
+  // if target element is a hidden one return null
+  const element = React.useMemo(() => {
+    const highlightElement = document.querySelector(selector);
+    let hiddenElement = highlightElement;
+    while (hiddenElement) {
+      const ariaHidden = hiddenElement?.getAttribute('aria-hidden');
```

Comment:
Same suggestion as last time, which wasn't applied:
```suggestion
      const ariaHidden = hiddenElement.getAttribute('aria-hidden');
```

Reply:
updated